assets/src/edit-story/components/panels/sizePosition.js

```diff
@@ -131,6 +131,12 @@ function SizePositionPanel({
   );
 
   usePresubmitHandler(({ rotationAngle: newRotationAngle }) => {
+    const isRotationAngle = Boolean(newRotationAngle);
+
+    if (!isRotationAngle) {
+      return null;
```

Comment:
Nit: this is correct, but feels a bit fragile because it will trigger also on `newRotationAngle === 0`. Perhaps we can keep this as is. Or we can document it in comments that 0 value is expected here and doesn't need update. Or even simpler, we can send `0` to the code below, which works just as well for zeros.